content/examples/cards.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Cards
+lastUpdatedDate: 2022-05-17T12:00:00.000Z
+---
+
+<div className="su-max-w-1200">
```

Comment:
Can you update this max-width to `lg:su-max-w-600 xl:su-max-w-800 2xl:su-max-w-1000`? That should address the code block width, as well as constrain the width of the card examples.